open.go

```diff
@@ -680,10 +680,17 @@ func GetVersion(dir string, fs vfs.FS) (string, error) {
 	return version, nil
 }
 
-// replayWAL replays the edits in the specified log file. If the DB is in
-// read only mode, then the WALs are replayed into memtables and not flushed. If
-// the DB is not in read only mode, then the contents of the WAL are guaranteed
-// to be flushed.
+// replayWAL replays the edits in the specified log file. If the DB is in read
+// only mode, then the WALs are replayed into memtables and not flushed. If
```

Comment:
How is read-only mode used? Is it mostly informational?

Do we need to guarantee that reads returned by read-only mode are durable? If so, can we [optionally?] fsync the WAL in read-only mode (even though it's read-only)?